packages/azure-kusto-ingest/test/e2eTests/e2eTest.ts

```diff
@@ -117,20 +119,20 @@ const main = (): void => {
                 try {
                     await queryClient.execute(databaseName, `.create-or-alter table ${tableName} ingestion json mapping '${mappingName}' '${mapping}'`);
                 } catch (err) {
-                    assert.fail("Failed to create table ingestion mapping, error: " + JSON.stringify(err));
+                    assert.fail("Failed to create table ingestion mapping, error: " + util.format(err));
                 }
             } catch (err) {
                 console.log(`Creating table ${tableName}, with columns ${tableColumns}`);
 
-                assert.fail(`Failed to create table ${tableName} ${err} ${databaseName}, error: ${JSON.stringify(err)}`);
+                assert.fail(`Failed to create table ${tableName} ${err} ${databaseName}, error: ${util.format(err)}`);
```

Comment:
What? you put an object into "util.format"? what does it even do? what if err is not a string? what if it happens to contain a %? why this change? JSON.stringify should be available everywhere